.vscode/settings.json

```diff
@@ -3,7 +3,8 @@
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
     // Don't want to use isort because it conflicts with Ruff - see run on save below
-    "source.organizeImports": false
+    "source.organizeImports": false,
```

Comment:
We might be able to make this true if isort doesn't get run anymore with the default formatter change